cmd/lkecli/main.go

```diff
@@ -135,8 +136,15 @@ func (k *kubeconfigCmd) Run(cfg *cfg) error {
 		return err
 	}
 
+	var contextName string
+	if k.Rename != "" {
+		contextName = k.Rename
```

Comment:
We could rename it by default to the cluster name?

Reply:
Yeah I like that, so still leave `name:"rename" default:""` but than modify that in the code to default to the cluster arg. Or is there a way to do dependent defaults with this cli library?

Reply:
Yeah - so this bit becomes `contextName = (k.Rename || cluster-pretty-name)`